server/src/main/xsd/gateway-config-201409.xsd

```diff
@@ -1225,6 +1229,18 @@
             <element name="ws.inactivity.timeout" maxOccurs="1" minOccurs="0" type="gateway:TimeIntervalString">
                 <annotation/>
             </element>
+            <element name="http.keepalive" maxOccurs="1" minOccurs="0">
```

Comment:
Default is enabled, right? I expect you are already on to this but please ensure this gets documented